Source/WebCore/platform/mediastream/mac/AVVideoCaptureSource.h

```diff
@@ -194,7 +194,7 @@ class AVVideoCaptureSource : public RealtimeVideoCaptureSource, private Orientat
 
 #if PLATFORM(IOS_FAMILY)
     bool m_shouldCallNotifyMutedChange { false };
-    Timer m_startupTimer;
+    std::unique_ptr<Timer> m_startupTimer;
```

Comment:
hm... EventLoopTimer will have this exact semantics but it's probably a layering violation to use that in WebCore/platform.